source/faq/questions/dashboard-faq.md

```diff
@@ -82,6 +82,12 @@ We have already begun the implementation for capturing even more things from you
 
 These will be added in subsequent releases.
 
+## {% fa fa-angle-right %} Why are my video recordings freezing or dropping frames when running in CI?
+
+There is a {% url 'known issue' https://github.com/cypress-io/cypress/issues/4722 %} where videos recorded to the Dashboard may freeze up or drop frames if there are not enough resources available when running the tests in your CI container. You can run your tests with {% url 'memory and CPU logs enabled' troubleshooting#Log-memory-and-CPU-usage %} to identify and evaluate the resource utilization within your CI.
```

Comment:
instead of GH issue URL, just put its number, see https://github.com/cypress-io/cypress-documentation/blob/develop/TAGS.md#issue

Reply:
This phrasing makes this sound like this is a Cypress problem - like a bug, but this is just how it works. I wouldn't link to this issue - this issue is closed as a 'wontfix'. 

The answer should be phrased as - you need enough CPU to run Cypress and record video or else CPU will lock up you running applications and freeze. Like, if you ran Google Chrome on a machine that did not have enough CPU to run Chrome, Chrome will lock up! This isn't a Chrome issue, you need the required CPU to run any application.